pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -1504,6 +1511,26 @@ def create_cincinnati_prs(self, assembly_type, release_info):
             self.skip_ota_notification
         )
 
+    def ocp_doomsday_backup(self):
+        """
+        Backup the payload to ocp-doomsday-registry s3 bucket on AWS. This function will trigger a pipeline on
+        ART cluster
+
+        :param major: Eg. 4.15
+        :param version: Eg. 4.15.10
+        """
+        data = {
+            "params": {
+                "major": self.group,
+                "version": self.assembly
```

Comment:
we promote ECs and RCs too, is anything blocking those from getting here?

Reply:
Added [here](https://github.com/openshift-eng/art-tools/pull/332/files#diff-de01749278991bbde16a03048265c175ddc0aaf9a68f3f878396c59b821e4573R444)